wire/internal/wire/wire_test.go

```diff
@@ -542,3 +546,23 @@ func (test *testCase) materialize(gopath string) error {
 	}
 	return nil
 }
+
+// readFile reads data from path. It uses io.Scanner to read the lines of
```

Comment:
Use of `bufio.Scanner` is more complex than using `bytes.Replace`. I would recommend the latter.